galley/cmd/galley/cmd/probe.go

```diff
@@ -37,7 +37,7 @@ func probeCmd() *cobra.Command {
 			if err := probe.NewFileClient(&probeOptions).GetStatus(); err != nil {
 				return fmt.Errorf("fail on inspecting path %s: %v", probeOptions.Path, err)
 			}
-			fmt.Println("OK")
+			_, _ = fmt.Fprintf(cmd.OutOrStdout(), "OK")
```

Comment:
You can leave out the `_, _ = ` here. (I'm surprised the linter isn't complaining that you didn't).

Reply:
Sure, I fixed it. This `_,_ =` style is used in many packages in Istio repository. So, I guessed that it was a convention or something (But it looks like I was wrong.)

e.g
https://github.com/istio/istio/blob/master/istioctl/pkg/install/verify.go#L159-L161